pkgs/development/python-modules/langgraph-checkpoint/default.nix

```diff
@@ -31,11 +33,16 @@ buildPythonPackage rec {
 
   dependencies = [ langchain-core ];
 
+  propagatedBuildInputs = [ msgpack ];
+
+  pythonRelaxDeps = [ "msgpack" ]; # Can drop after 1.1.0 release
```

Comment:
Is this comment still relevant now that version 2 is available?

Reply:
Sorry, I meant msgpack 1.10.0

Changing msgpack has to go through staging (2800+ affected)

Reply:
Should I comment around bedtime? Clearly not. In any case, I clarified the comment text. 

Thanks for catching it.

Reply:
Thanks 🙏